Jaron Cheung's Project #1 Submission

JaronJCheung opened the pull request:
Dear Professor @johnbeve,

My Project 1 is complete! I have created a new pull request because I tried syncing my main branch with the updated files of your main and it deleted the work on my Project 1 even though I changed the file name. Karl and I, however, figured out how to rebase our main branches from Github desktop. Please let me know if you need anything else from me.

Thanks and take care,
Jaron Cheung

JaronJCheung committed:
Rename README.md to JCheung-P1-README.md

johnbeve commented:
@JaronJCheung There doesn't seem to be anything here...

<img width="1300" alt="Screen Shot 2023-02-16 at 3 01 20 PM" src="https://user-images.githubusercontent.com/16356971/219474095-8812f2df-5992-47e2-9a6f-7d673ad5b2ef.png">

See in the screenshot there is a green +0 and red -0, indicating nothing has been added or subtracted.

JaronJCheung commented:
> @JaronJCheung There doesn't seem to be anything here...
> 
> <img alt="Screen Shot 2023-02-16 at 3 01 20 PM" width="1300" src="https://user-images.githubusercontent.com/16356971/219474095-8812f2df-5992-47e2-9a6f-7d673ad5b2ef.png">
> 
> See in the screenshot there is a green +0 and red -0, indicating nothing has been added or subtracted.

@johnbeve, ahh this is unfortunate. My apologies. I will work on getting this resolved ASAP. 

This happened because I pressed sync for my main repository so that I could update it alongside the changes you made in your repository. Despite the fact that I had changed the name of the branch that I had rebased via Git Desktop, my renamed Project 1 was wiped. 

I will be attempting to merge or rebase my branch to my main repository by command line. I just need to learn how to do that. 

If you would like my work immediately, I can create a pull request from my branch, which is not my main repo. Is this something you would like?

Thanks, 
Jaron

johnbeve committed:
Create JCheung-P1-README.md

johnbeve commented:
Gotcha, thank you for explaining. I've copied the content on your branch to this pull request; everything seems to be there.

johnbeve commented on Project-1/JCheung-P1-README.md:
```diff
@@ -0,0 +1,474 @@
+
+(a)  B(x,y) that says that x is a brother of y:
+  
+**B(xy) = ∃z(P(z,x) ∧ P(z,y)) ∧ ~F(x)**
```
Let x=y, ~F(x), and P(z,x). Then B(x,x) follows, i.e. then someone is their own brother.

johnbeve commented on Project-1/JCheung-P1-README.md:
```diff
@@ -0,0 +1,474 @@
+
+(b)  A(x,y) that says that x is an aunt of y
+  
+**A(xy) = ∃z∃u(P(u,x) ∧ P(u,z) ∧ P(z,y) ∧ F(x))**
```
This definition allows x=y, y=z, etc. 

This definition doesn't represent aunts by marriage.

johnbeve commented on Project-1/JCheung-P1-README.md:
```diff
@@ -0,0 +1,474 @@
+
+**I am providing two possible sets of answers for (a)-(e) (I also go back and forth between using markdown code and not. My apologies for the inconsistency):**
+
+(a)  B that says that x is a brother of y
```
As mentioned in seminar, ALC doesn't have inverses or cardinalities.

johnbeve commented on Project-1/JCheung-P1-README.md:
```diff
@@ -0,0 +1,474 @@
+
+(c)  C(x,y) that says that x and y are cousins   
+  
+**C(xy) = ∃u∃w∃z(P(u,w) ∧ P(u,z) ∧ P(w,x) ∧ P(z,y) ∧ ~(z=u)) ∧ ~(x=y)**  
```
Counterexample: Second cousins

johnbeve commented on Project-1/JCheung-P1-README.md:
```diff
@@ -0,0 +1,474 @@
+
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+![proof](https://user-images.githubusercontent.com/123851163/216893568-701f0b47-bd92-43fe-9fe7-e6a26d95985a.png)
```
What justifies moving from a negated universal in the first line to a negated universal in the second?

johnbeve commented on Project-1/JCheung-P1-README.md:
````diff
@@ -0,0 +1,474 @@
+```
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+
+¬(Px ∧ Qx) → (¬Px ∨ ¬Qx)	
````
You did not instantiate the universal variable x to a constant. 

You did not provide any justification lines for your proof...

JaronJCheung committed:
Rename README.md to JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Update JCheung-P2-README.md

JaronJCheung committed:
Merge pull request #6 from JaronJCheung/JaronJCheung-P-2

Update JCheung-P2-README.md

JaronJCheung committed:
Update README.md

JaronJCheung committed:
Update README.md

JaronJCheung committed:
Update and rename README.md to JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Update JCheung-P3-README.md

JaronJCheung committed:
Merge pull request #8 from JaronJCheung/JaronJCheung-P-3

Update JCheung-P3-README.md

JaronJCheung committed:
Update README.md

JaronJCheung committed:
Rename README.md to JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

Dear @peihongx and @jonathanvajda,

You are invited to review my Project 4. This is unfinished at the moment. I will let you know when it is good for further review. Thank you for your consideration, time, and for the review! Please let me know if you have any questions.

Best,
Jaron

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Update JCheung-P4-README.md

JaronJCheung committed:
Merge pull request #9 from JaronJCheung/JaronJCheung-P-4

Update JCheung-P4-README.md

JaronJCheung committed:
Create Project-5.md

JaronJCheung committed:
Rename Project-5.md to Project-5-Task-1.md

JaronJCheung committed:
Update Project-5-Task-1.md

JaronJCheung committed:
Update Project-5-Task-1.md

JaronJCheung committed:
Update Project-5-Task-1.md

tmprd committed:
Sample SHACL shape + instance

tmprd committed:
Update BFO-SHACL-Instances.md

tmprd committed:
Update BFO-SHACL-Instances.md

JaronJCheung committed:
BFO 2020 Knowledge Graph with Instance Data Added

JaronJCheung committed:
BFO SHACL Schema 3.0 for BFO 2020 Knowledge Graph with Instance Data

JaronJCheung committed:
Update Project-5-Task-1.md

tmprd committed:
Merge branch 'JaronJCheung-P-5' into Tim-Project-5

tmprd committed:
Add some of my sense data

tmprd committed:
Try to fix weird spacing change

tmprd committed:
Fix IRI

tmprd committed:
Make some contradictions + add Berkley's mind

tmprd committed:
Update validations doc

tmprd committed:
Update instances + validation result

tmprd committed:
Update bad proposition

tmprd committed:
FOL Spatial Axioms

tmprd committed:
Try another axiom...

tmprd committed:
Merge pull request #11 from JaronJCheung/Tim-Project-5

Tim Project 5

JaronJCheung committed:
Update Project-5-Task-1.md

JaronJCheung committed:
Update Project-5-Task-1.md

JaronJCheung committed:
Rename BFO-SHACL-Instances.md to BFO-SHACL-Instances-Tim.md

JaronJCheung committed:
Rename BFO_2020_Knowledge Graph.ttl to 1. BFO_2020_Knowledge Graph.ttl

JaronJCheung committed:
Rename BFO_SHACL_SCHEMA_3.0.ttl to 2. BFO_SHACL_SCHEMA_3.0.ttl

JaronJCheung committed:
Rename Project-5-Task-1.md to 1. Project-5-Task-1.md

JaronJCheung committed:
Rename 1. BFO_2020_Knowledge Graph.ttl to 2. BFO_2020_Knowledge Graph.ttl

JaronJCheung committed:
Rename 2. BFO_SHACL_SCHEMA_3.0.ttl to 3. BFO_SHACL_SCHEMA_3.0.ttl

JaronJCheung committed:
Rename 2. BFO_2020_Knowledge Graph.ttl to 2. Project_5_BFO_2020_Knowledge Graph.ttl

JaronJCheung committed:
Rename 1. Project-5-Task-1.md to 1. Project_5_Task_1.md

JaronJCheung committed:
Rename 3. BFO_SHACL_SCHEMA_3.0.ttl to 3. Project_5_BFO_SHACL_SCHEMA_3.0.ttl

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Create Project-5

JaronJCheung committed:
Create P5

JaronJCheung committed:
Rename 1. Project_5_Task_1.md to Project-5/1. Project_5_Task_1.md

JaronJCheung committed:
Rename 2. Project_5_BFO_2020_Knowledge Graph.ttl to Project-5/2. Project_5_BFO_2020_Knowledge Graph.ttl

JaronJCheung committed:
Rename 3. Project_5_BFO_SHACL_SCHEMA_3.0.ttl to Project-5/3. Project_5_BFO_SHACL_SCHEMA_3.0.ttl

JaronJCheung committed:
Delete P5

JaronJCheung committed:
Delete Spatial-Axioms.ttl

JaronJCheung committed:
Delete directory directory

JaronJCheung committed:
Final BFO SHACL Schema with ALL SHACL Shapes Added (92)

JaronJCheung committed:
Rename FINAL_BFO_SHACL_Schema.ttl to 4. FINAL_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Special Collection Version BFO SHACL Schema

This is a delimited special collection of SHACL shapes from Karl, Giacomo, and Finn that returned 0 errors. However, there are violations that are returned on this knowledge graph. These violations are primarily centered on "min count 1" constraints.

JaronJCheung committed:
Rename SPECIAL_COLLECTION_BFO_SHACL_Shapes_Karl-Giacomo-Finn.ttl to 5. SPECIAL_COLLECTION_BFO_SHACL_Shapes_Karl-Giacomo-Finn.ttl

JaronJCheung committed:
Rename 4. FINAL_BFO_SHACL_Schema.ttl to 4. Project_5_FINAL_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Rename 5. SPECIAL_COLLECTION_BFO_SHACL_Shapes_Karl-Giacomo-Finn.ttl to 5. Project_5_SPECIAL_COLLECTION_BFO_SHACL_Shapes_Karl-Giacomo-Finn.ttl

JaronJCheung committed:
Rename BFO-SHACL-Instances-Tim.md to Project-5/BFO-SHACL-Instances-Tim.md

JaronJCheung committed:
Create BFO-SHACL-Instances-Tim.md

JaronJCheung committed:
Delete BFO-SHACL-Instances-Tim.md

JaronJCheung committed:
Rename 3. Project_5_BFO_SHACL_SCHEMA_3.0.ttl to 3. Project_5_TEMPLATE_SHACL_SCHEMA_3.0.ttl

JaronJCheung committed:
Rename 3. Project_5_TEMPLATE_SHACL_SCHEMA_3.0.ttl to 3. Project_5_TEMPLATE_BFO_SHACL_SCHEMA_3.0.ttl

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Rename 5. Project_5_SPECIAL_COLLECTION_BFO_SHACL_Shapes_Karl-Giacomo-Finn.ttl to 5. Project_5_Special_Collection_BFO_SHACL_Shapes_Karl-Giacomo-Finn.ttl

JaronJCheung committed:
Delete 4. Project_5_FINAL_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Add files via upload

JaronJCheung committed:
Rename Project_5_FINAL_BFO_SHACL_Schema.ttl to 4. Project_5_FINAL_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Rename 3. Project_5_TEMPLATE_BFO_SHACL_SCHEMA_3.0.ttl to 3. Project_5_TEMPLATE_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Rename 3. Project_5_TEMPLATE_BFO_SHACL_Schema.ttl to 3. Project_5_Template_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Rename 4. Project_5_FINAL_BFO_SHACL_Schema.ttl to 4. Project_5_BFO_SHACL_Schema_FINAL.ttl

JaronJCheung committed:
Rename 4. Project_5_BFO_SHACL_Schema_FINAL.ttl to 4. Project_5_FINAL_BFO_SHACL_Schema.ttl

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Update 1. Project_5_Task_1.md

JaronJCheung committed:
Merge pull request #10 from JaronJCheung/JaronJCheung-P-5

Update Project-5-Task-1.md
